app/views/admin/facilities/show.html.erb

```diff
@@ -36,18 +36,25 @@
       <% end %>
     </div>
   </div>
-  <div class="col-md-4">
+  <div class="col-md-3">
     <div class="card">
       <h3>Estimated OPD load</h3>
       <div><%= @facility.monthly_estimated_opd_load || "(N/A)" %> adult patients per month</div>
     </div>
   </div>
-  <div class="col-md-4">
+  <div class="col-md-3">
     <div class="card">
       <h3>Facility type &amp; size</h3>
       <%= @facility.facility_type || "(No type set)" %> (<%= @facility.facility_size&.capitalize || "No size set" %>)
     </div>
   </div>
+  <div class="col-md-3">
+    <div class="card">
+      <h3>Features</h3>
+      Diabetes Management: <%= @facility.diabetes_enabled? ? "Enabled" : "Disabled"  %>
+      Teleconsultation:    <%= @facility.teleconsultation_enabled? ? "Enabled" : "Disabled" %>
```

Comment:
Can we add the teleconsult phone numbers to the card too?

Reply:
Wouldn't look good. This card is already too big. I'll line these cards up evenly, but if we want numbers we should make this a larger card in a different row.

![Screenshot 2020-06-17 at 6 28 45 PM](https://user-images.githubusercontent.com/48791115/84901085-98b26f80-b0c8-11ea-86eb-04ff760c4347.png)

Reply:
Oh, annoying 😓 Ok, let' just stick to the two flags for now, awesome.

Is there a quick flex trick to make the four cards even? 😅 If not, no big deal.